.env.example

```diff
@@ -39,4 +39,12 @@ DB_SERVER=postgres
 POSTGRES_DB=reunice
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=root
+#############################################
+
+# Configuration of the smtp server
+#############################################
+SMTP_SERVER=mailpit
+SMTP_PORT=1025
+SMTP_USERNAME=test
+SMTP_PASSWORD=test

```

Comment:
@BartoszDolata to są zmienne środowiskowe, które możesz wykorzystać w aplikacji dla serwisu od wysyłania maili